docs/changelog.rst

```diff
@@ -2,6 +2,7 @@
 Changelog
 =========
 
+* :bug:`-` AdEx and Uniswap modules won't query events and trades prior the end date of the last used query range.
```

Comment:
You can remove the changelog though. This is not something visible by the user and they don't really know or can see the used query range concept.

Reply:
removed